packages/node-core/src/indexer/testing.service.ts

```diff
@@ -216,10 +231,54 @@ export abstract class TestingService<A, SA, B, DS> {
       this.totalFailedTests += test.expectedEntities.length;
       logger.warn(e, `Test ${test.name} failed to run`);
     } finally {
+      await this.dropSchema(schema);
+    }
+  }
+
+  private async dropSchema(schema: string) {
+    const modelsRelation = getAllEntitiesRelations(this.project.schema);
+
+    try {
+      // drop existing project schema and metadata table
       await this.sequelize.dropSchema(`"${schema}"`, {
         logging: false,
         benchmark: false,
       });
+
+      // TODO, remove this soon, once original enum are cleaned
+      // Deprecate, now enums are moved under schema, drop schema will remove project enums
+      await Promise.all(
```

Comment:
I think all new projects should be creating enums under the schema. Seeing as the testing data is ephemeral I don't see why enums should ever be under public.